<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskerUIMainWindowQWidget</class>
 <widget class="QWidget" name="TaskerUIMainWindowQWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasker</string>
  </property>
  <widget class="QFrame" name="welcomeQFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="makeCommitmentQPushButton">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>50</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Make Commitment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quickSessionQPushButton">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>50</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quick Session</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcomeDescriptionQLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>You have no commitments at this time.</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
